app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/AcceptOfferDialog.scala

```diff
@@ -12,15 +13,27 @@ import scalafx.scene.control._
 import scalafx.scene.layout._
 import scalafx.stage.Window
 
+import java.net.{InetSocketAddress, URI}
 import scala.collection._
 import scala.util.{Failure, Success, Try}
 
-class AcceptOfferDialog extends CliCommandProducer[AcceptDLCOffer] {
+class AcceptOfferDialog extends CliCommandProducer[AcceptDLCCliCommand] {
 
-  override def getCliCommand(): AcceptDLCOffer = {
+  override def getCliCommand(): AcceptDLCCliCommand = {
     val offerHex = offerTLVTF.text.value
     val offer = LnMessageFactory(DLCOfferTLV).fromHex(offerHex)
-    AcceptDLCOffer(offer)
+    if (peerAddressTF.text.value.nonEmpty) {
+      val peer = new URI(
+        "tcp://" + NetworkUtil.parseInetSocketAddress(peerAddressTF.text.value,
+                                                      2862))
```

Comment:
Do we want to check if the user entered a custom port? It seems to me that we should default to `2862` (maybe put this in a constant somewhere?) if they didn't add a port into the text box?

Reply:
It does, 2862 is the default if they don't enter one

Reply:
I am not sure where we would put the constant, the gui module doesn't have access to the DLC node stuff